test/subarray.jl

```diff
@@ -89,53 +89,50 @@ function single_stride_dim(A::Array)
 end
 single_stride_dim(A::AbstractArray) = single_stride_dim(copy_to_array(A))
 
-# Extract the "linear indexing dimension" from a SubArray
-getLD{T,N,P,I,LD}(::SubArray{T,N,P,I,LD}) = LD
-
-# Compare the linear indexing dimension of a SubArray
-# to a direct computation of strides
-function cmpLD(Atest::SubArray, Acomp)
-    # Compute ld, skipping over dropped dimensions
-    LD = getLD(Atest)
-    ld = LD
-    for i = 1:LD
-        if isa(Atest.indexes[i], Real)
-            ld -= 1
-        end
-    end
-    ld, single_stride_dim(Acomp)
-end
-
-# Testing linear dimension inference for views-of-views
-for N = 1:4
-    @eval begin
-        function test_viewview{T}(SB, A::Array{T,$N}, f, vindex)
-            local SSB
-            @nloops $N j d->(1:length(vindex)) d->(i_d = vindex[j_d]) begin
-                I = @ntuple $N d->i_d
-                try
-                    SSB = f(SB, I...)
-                catch err
-                    println(summary(SB))
-                    println(I)
-                    rethrow(err)
-                end
-                SA = f(A, I...)
-                ld, ldc = cmpLD(SSB, SA)
-                if ld == ldc
-                elseif ld <= ldc
-                    if print_underestimates
-                        println("Underestimate f = ", f, " on ", summary(SB), " with I = ", I, ", producing ", summary(SSB))
-                    end
-                else
-                    println(summary(SB))
-                    println(summary(SSB))
-                    error("failed on ", I)
-                end
-            end
-        end
-    end
-end
+# # Compare the linear indexing dimension of a SubArray
```

Comment:
Ah, I forgot about this chunk.  Yes, it can be deleted — it's intrinsically tied to the meaning of the old `LD` parameter, which was pretty complicated.